packages/driver/src/cypress/error_utils.ts

```diff
@@ -572,6 +574,25 @@ const logError = (Cypress, handlerType: HandlerType, err, handled = false) => {
   })
 }
 
+const isLogabbleError = (error: unknown): error is Error => {
+  return (
+    typeof error === 'object' &&
+    error !== null &&
+    'message' in error &&
+    typeof (error as Record<string, unknown>).message === 'string'
```

Comment:
Do we need such an aggressive check? Would `_.isObject(error)` be sufficient?

Reply:
we also need to verify if message is a string